source/style-guide/style/metadata.txt

```diff
@@ -0,0 +1,19 @@
+.. _metadata:
+
+=============
+Metadata
+=============
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Use ``.. meta::`` to add meta tags to a page. Adding meta tags aids in 
+searches both internally and via the Internet.
```

Comment:
Can we go into a bit more detail about what "aids in searches" means? For example, perhaps we could add a sentence that says something like:

"When users search for terms defined in meta tags, pages with those tags are given higher priority in search results."

Note that the wording could probably be improved here, but I think something that gets a bit more specific about how these tags impact searches would be helpful.